test/e2e/scheduling/ubernetes_lite.go

```diff
@@ -52,6 +54,20 @@ var _ = SIGDescribe("Multi-AZ Clusters", func() {
 		msg := fmt.Sprintf("Zone count is %d, only run for multi-zone clusters, skipping test", zoneCount)
 		e2eskipper.SkipUnlessAtLeast(zoneCount, 2, msg)
 		// TODO: SkipUnlessDefaultScheduler() // Non-default schedulers might not spread
+
+		cs := f.ClientSet
+		e2enode.WaitForTotalHealthy(cs, time.Minute)
+		nodeList, err := e2enode.GetReadySchedulableNodes(cs)
+		framework.ExpectNoError(err)
+
+		// make the nodes have balanced cpu,mem usage
+		cleanUp, err = createBalancedPodForNodes(f, cs, f.Namespace.Name, nodeList.Items, podRequestedResource, 0.6)
```

Comment:
Why 0.6? I understand that in the preemption test we needed to leave little space so that preemption is forced, but we don't need 0.6 for spreading right? Reading the source code, passing 0.0 will make createBalancedPodForNodes balance all nodes with the same ratio as the max ratio amongst all nodes, which should be sufficient right?